bundles/org.openhab.binding.bloomsky/p

```diff
@@ -0,0 +1 @@
+[INFO] Scanning for projects...
```

Comment:
I think this should be removed from your PR

Reply:
Yes, and removed.